chain/events/filter/index.go

```diff
@@ -266,20 +274,7 @@ func (ei *EventIndex) migrateToVersion2(ctx context.Context, chainStore *store.C
 		return xerrors.Errorf("commit transaction: %w", err)
 	}
 
-	// during the migration, we have likely increased the WAL size a lot, so lets do some
```

Comment:
I removed the vacuum entirely from this migration, leaving it only in the last migration so we only do one rewrite at the very end